_pytest/main.py

```diff
@@ -177,6 +190,15 @@ def pytest_ignore_collect(path, config):
     if py.path.local(path) in ignore_paths:
         return True
 
+    allow_in_venv = config.getoption("collect_in_virtualenv")
+    if _in_venv(path) and not allow_in_venv:
```

Comment:
I'm not sure we should emit this warning; the common case is for users to have some virtual environment locally, so we would generate needless warnings all the time.